debug/gdbserver.py

```diff
@@ -291,17 +300,22 @@ class MemTestBlock(GdbTest):
     length = 1024
     line_length = 16
 
-    def test(self):
-        a = tempfile.NamedTemporaryFile(suffix=".ihex")
+    def write(self, temporary_file):
         data = ""
         for i in range(self.length / self.line_length):
             line_data = "".join(["%c" % random.randrange(256)
                 for _ in range(self.line_length)])
             data += line_data
-            a.write(ihex_line(i * self.line_length, 0, line_data))
-        a.flush()
+            temporary_file.write(ihex_line(i * self.line_length, 0, line_data))
+        temporary_file.flush()
+        return data
+
+    def test(self):
+        a = tempfile.NamedTemporaryFile(suffix=".ihex")
+        data = self.write(a)
 
         self.gdb.command("shell cat %s" % a.name)
+        self.gdb.command("monitor riscv reset_delays 50")
```

Comment:
Is reset_delays a new subcommand in openocd?

Reply:
Yes, added in https://github.com/riscv/riscv-openocd/pull/344 (which hasn't merged yet).